contracts/Ethereum_SpokePool.sol

```diff
@@ -32,7 +32,7 @@ contract Ethereum_SpokePool is SpokePool, Ownable {
         IERC20(relayerRefundLeaf.l2TokenAddress).safeTransfer(hubPool, relayerRefundLeaf.amountToReturn);
     }
 
-    // Admin is simply owner which should be same account that owns the HubPool deployed on this network. A core
-    // assumption of this contract system is that the HubPool is deployed on Ethereum.
+    // The SpokePool deployed to the same network as the HubPool must be owned by the HubPool.
+    // A core assumption of this contract system is that the HubPool is deployed on Ethereum.
```

Comment:
I think this is a good place to mention that `crossDomainAdmin` is also unused for this spoke pool and arbitrarily set to the deployer. There is no harm leaving it as the deployer

Reply:
> I think this is a good place to mention that `crossDomainAdmin` is also unused for this spoke pool and arbitrarily set to the deployer. There is no harm leaving it as the deployer

Any thoughts on [this](https://github.com/across-protocol/contracts-v2/pull/217/commits/3be288a773d1106376d9343669423b1b6c23a971) instead?

Reply:
yea thats more consistent but still add a comment why we're setting `crossDomainAdmin` to that